zerver/webhooks/discourse/doc.md

```diff
@@ -0,0 +1,21 @@
+On your {{ settings_html|safe }},
+[create a bot](/help/add-a-bot-or-integration) for
+{{ integration_display_name }}. Make sure that you select
+**Incoming webhook** as the **Bot type**:
+
+![](/static/images/help/bot_types.png)
+
+The API keys for "Incoming webhook" bots are limited to only
+sending messages via webhooks. Thus, this bot type lessens
+the security risks associated with exposing the bot's API
+key to third-party services.
+
+Copy the `API KEY` provided - you'll need it later.
+
+Head over to the
+[Discourse Chat Integration Setup Instructions](https://meta.discourse.org/t/68501)
+and complete them.
+
+{!congrats.md!}
+
+![](/static/images/integrations/discourse/001.png)
```

Comment:
I didn't include the create-stream macro, as there there isn't a requirement to do so. The integration is designed so it can post into any existing stream.

Re. the macros - I started with the create-stream macro but found some stuff wasn't particularly relevent. Is there anything I need to do for the purpose of this PR, or is it ok to leave as-is?